### PR TITLE
feat: proclaim to the world that we are indeed powered by Vercel!

### PR DESCRIPTION
**Pull request by ob6160**
## What does this change?
Prerequisites to applying for an open source project sponsorship from Vercel 🔺 

* Add the Vercel logo to the footer and reinstate it on mobile
* Add the Vercel logo to our readme in a prominent location
* Link back to Vercel with our organisation name as the slug

## Screenshots

*Desktop*

<details>
<summary>Details</summary>

gbptm-git-ob-powered-by-vercel-neontribe vercel app_

</details>

*Mobile*

<details>
<summary>Details</summary>

gbptm-git-ob-powered-by-vercel-neontribe vercel app_(iPhone SE)

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 This documentation is oriented towards developers, if you'd like to learn more about our data and how to access it please refer to [Toilet Map Explorer](https://www.toiletmap.org.uk/explorer).
 
+![Powered by Vercel](./public/powered-by-vercel.svg)
```
Comment (marxian) [inline README.md:7]: Should this be a link back to vercel too?

File: src/components/LooMap/LooMap.tsx [hunk 2]
```diff
@@ -279,7 +279,7 @@ const LooMap: React.FC<LooMapProps> = ({
         `}
       >
         <TileLayer
-          attribution="&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a> contributors"
+          attribution="&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a> contributors | <a href='https://vercel.com/?utm_source=public-convenience-ltd&utm_campaign=oss'>Powered by Vercel &#9650;</a>"
```
Comment (marxian) [inline src/components/LooMap/LooMap.tsx:282]: This is weird though. This attribution refers to the map tiles. Do you think they really meant this?

File: README.md (hunk 1 above)
Comment (ob6160) [reply README.md:7]: agreed, that would make sense

File: src/components/LooMap/LooMap.tsx (hunk 2 above)
Comment (ob6160) [reply src/components/LooMap/LooMap.tsx]: as discussed, removed this and now reinstated a small mobile footer

**Review by marxian — APPROVED** [review]

**Pull request merged**